### ISSUE TITLE
Needs comments section

### ISSUE THREAD

**Issue by zbecknell** [messman/of-the-day#1]
And/or, a simple +/- system so each day can be ranked.  That being said, both of these things benefit from some sort of auth system, so that's quite a lot more to add in.

**Comment by messman** [comment]
Hey man! Yes, the page is read-only with no auth right now, and adding those features would require a more robust data storage solution and auth. 
I'll look into a simple version that maybe only allows pre-defined responses or requires responses to be approved. Thanks for the suggestion!

**Comment by zbecknell** [comment]
Good job deleting Facebook!  I've been Facebook-free many years now... it was a nice way of keeping contacts, but that doesn't justify all its downsides.

**Comment by messman** [comment]
Thanks! I'm realizing now it's a little more difficult than I thought to not have that same 'default' form of communication to use with people - evidenced here with our conversation through GitHub Issues 😄 

I know it's been awhile since I have looked at the Of-The-Day... I was busy with [Tidy](https://github.com/messman/tidy). Good news is, this project is no longer the lowest programming priority - it's now tied for highest with Tidy.

I'm still thinking it can be a minimalist comment section and use a basic auth system. That might not take forever to do and it's something I've wanted to learn more about anyway.

**Comment by zbecknell** [comment]
Yeah, I saw you got your family to use Slack -- good idea!  Comments certainly would be nice here, but actually it'd be really nice to see all the posts (another priority I know) since you have some really good content and sometimes I want to go back and look at what it was.

So don't worry about comments if I'm the only one that expressed interest!

Be safe out there!  Two things I've been enjoying lately are a new book by Erik Larson: https://www.amazon.com/Splendid-Vile-Churchill-Family-Defiance-ebook/dp/B07TRVW6VX/ref=sr_1_1?crid=1E8NH3WCQNY6D&dchild=1&keywords=the+splendid+and+the+vile+by+erik+larson&qid=1590618028&sprefix=the+splendid+%2Caps%2C215&sr=8-1

As I told our random channel:

> If you like books, give _The Splendid and the Vile: A Saga of Churchill, Family, and Defiance During the Blitz_ by Erik Larson a chance.  Whether you prefer fiction or non-fiction, this will fit the bill.  Erik Larson puts true stories together in a highly compelling manner evocative of the best novels.  Whether or not you've ever been interested in Churchill before, I highly recommend this book.  The cast of (true) characters and the events that take place would be hard to make up if they weren't real.

And the other: https://www.montydon.com/

He's Britain's favorite gardener and a great/enthusiastic presenter.  You can view most of his shows on a site called HD Clump.  Great relaxing material or if any of your family is into gardening, he's very informative.

Oh!  Hope you saw that John Oliver is sponsoring Marbula 1!

**Comment by messman** [comment]
Well, well, well! Almost 5 months later, I'm coming back around to respond to this one-and-only (for now) issue. I hope you've been well, @zbecknell! I'll message you on LinkedIn. Thanks for those book suggestions by the way!

> but actually it'd be really nice to see all the posts (another priority I know) since you have some really good content and sometimes I want to go back and look at what it was

This part is now handled! See the 'Archive' feature at [oftheday.andrewmessier.com/archive](https://oftheday.andrewmessier.com/archive). It's not perfect (no search, no pagination, no load-as-I-scroll), but I think it gets the job done. Thanks for your request! By the time I finished this revision, I think the archive section was the most-requested feature. 

> So don't worry about comments if I'm the only one that expressed interest!

Comments have also been suggested multiple times, so you aren't the only one. At this point, however, I don't think I'll be putting the effort toward it. As I now have covered in the 'About' page:

> For now, no. I think they would be cool additions, but I would prefer to let the dust settle on this project for awhile. I am also opposed to this site becoming too much like a social media site, as that is not its purpose.

As you may have seen if you've been watching the page, I've really been devoting a lot of time to this project because I'm a functioning workaholic. It's going to take a lot of my energy just to step away from it for the next week instead of rewriting the mobile view.

But! Another project is on the horizon, and we'll see what that has in store. It will definitely need an auth system, so maybe it would be easy to back-port it to this project.

Thanks for bringing up this issue in the first place. I hope everyone is well at my favorite company. 

If you see any other issues, throw 'em up for me to see. Cheers!
